shared/chat/conversation/notices/profile-reset-notice/container.js

```diff
@@ -17,7 +17,7 @@ const mapStateToProps = (state: TypedState) => {
 
   return {
     prevConversationIDKey: supersedes.conversationIDKey,
-    username: supersedes.finalizeInfo.resetUser,
+    username: supersedes.finalizeInfo && supersedes.finalizeInfo.resetUser,
```

Comment:
Hmm what was the crasher? flow-types should mean that finalizeInfo can't be null, right?

Reply:
This was the stacktrace @mmaxim saw: 

![image](https://user-images.githubusercontent.com/11968340/32071616-d1f3e1c0-ba5d-11e7-99e5-104ac33b01d6.png)

My hunch is that this is a race with `mapStateToProps` being called with a different `selectedConversationIDKey`

Reply:
Hmm...the stack trace refers to `old-profile-reset-notice`, but you're working with `container.js` in `profile-reset-notice`. Looking at that line, it may have already been fixed by one of @chrisnojima's recent PRs.

Reply:
@akalin-keybase Oh, whoops! I completely glanced over the difference. And it looks like new flow caught that one in Nojima's PR, cool! I'll close this one out.